psycopg_c/pyproject.toml

```
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools >= 49.2.0", "wheel >= 0.37", "tomli >= 2.0.1"]
+requires = ["setuptools >= 49.2.0", "wheel >= 0.37", "tomli >= 2.0.1 ; python_version<'3.11'"]
```

Thank you! Can you please split this list on multiple lines?

```suggestion
requires = [
    "setuptools >= 49.2.0",
    "wheel >= 0.37",
    "tomli >= 2.0.1 ; python_version<'3.11'",
]
```